openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config_test.go

```diff
@@ -0,0 +1,287 @@
+package node
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	configv1 "github.com/openshift/api/config/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+)
+
+func TestValidateConfigNodeForExtremeLatencyProfile(t *testing.T) {
```

Comment:
I assume that this test was moved from the previous file and wasn't changed.

Reply:
indeed!